### PR TITLE
Feature readme

### PR DESCRIPTION
**Pull request by ArthurFleischman**
## adding README.md content, such as:
- setup
> - install python
> - install pip
> - install venv
> - install libs
- use VENV(activate/deactivate)
- run server

### CODE REVIEW

File: venv/bin/activate [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    if [ ! "${1:-}" = "nondestructive" ] ; then
+    # Self destruct!
+        unset -f deactivate
+    fi
+}
+
+# unset irrelevant variables
+deactivate nondestructive
+
+VIRTUAL_ENV="/home/arthur/VisualStudioProjects/Python/ranked-voting/venv"
```
Comment (gegen07) [inline venv/bin/activate]: The `VIRTUAL_ENV` is hardcoded. It won't work in other paths like mine, I think.

Comment (ArthurFleischman) [reply venv/bin/activate]: oh, ok it may not be the best or im not knowing how to set it up for everyone

Comment (ArthurFleischman) [reply venv/bin/activate]: although, thanks.

Comment (ArthurFleischman) [reply venv/bin/activate]: ill try to use the relative path, or ill delete it.

Comment (ArthurFleischman) [reply venv/bin/activate]: i updated to relative path... still working

Comment (gegen07) [reply venv/bin/activate]: I think if you create a `requirements.txt` and explain how to use a virtualenv with this file it would be better. :smile:

Comment (ArthurFleischman) [reply venv/bin/activate]: ok, ill do it latter

Comment (ArthurFleischman) [reply venv/bin/activate]: but is a common environment which allow everybody have the same packages, looks like a docker container, i putted the "how to use" in readme.md

Comment (ArthurFleischman) [reply venv/bin/activate]: so, everything that i install in our "common python interpreter" will be available for everyone INSIDE the VENV, it means nobody ill have to config the setup, resulting in less missing time, helping people to fix broken installs

Comment (ArthurFleischman) [reply venv/bin/activate]: try it on your machine, clone my fork read the readme, and try running it

**Comment by lucasmontano** [comment]
@ArthurFleischman I won't be able to merge this PR, I still see a lot of changes not related to the ReadME :/

**Comment by ArthurFleischman** [comment]
My bad, bad habbit

**Comment by ArthurFleischman** [comment]
You are helping me a lot... thank you

**Comment by ArthurFleischman** [comment]
I'll delete de venv, by the way, it's all generated files.... I'm not adding 42 files...

**Comment by ArthurFleischman** [comment]
now it may be ok

**Review by gegen07 — APPROVED** [review]

**Pull request closed without merging**